app/src/components/integrations/containers/integrationInfoContainer/integrationInfoContainer.jsx

```diff
@@ -70,7 +69,7 @@ export class IntegrationInfoContainer extends Component {
       showToggleConfirmationModal,
       events,
     } = this.props;
-    const pluginTitle = PLUGIN_NAME_TITLES[name] || name;
+    const pluginTitle = details.name || name;
```

Comment:
Do we need to keep usage of `PLUGIN_NAME_TITLES` as a fallback for backward compatibility with existing plugins?
Do you left the `PLUGIN_NAME_TITLES` constant and its content untouched intentionally?

Reply:
No, we don't, all built-in plugins have display name in the develop version.  Existing plugins will be updated.
Other parts of the code still use PLUGIN_NAME_TITLES, I didn't change it because it's not clear to me.